client-tools/Shell/ShellFeature.cpp

```diff
@@ -80,6 +83,10 @@ void ShellFeature::collectOptions(
 
   options->addOption("--javascript.run-main", "Execute main function.",
                      new BooleanParameter(&_runMain));
+
+  options->addOption("--telemetrics.send-to-endpoint",
+                     "Send telemetrics to endpoint",
+                     new BooleanParameter(&_sendToEndpoint));
```

Comment:
If this option is only useful for testing purposes, shouldn't we
1. mark it as Uncommon
2. only enable it in maintainer mode
?